Cargo.toml

```diff
@@ -34,12 +36,23 @@ holochain_types = "0.3.0-beta-dev.31"
 holochain_nonce = "0.3.0-beta-dev.22"
 holochain_zome_types = { workspace = true }
 
-lair_keystore_api = "0.4.0"
+lair_keystore_api = { version = "0.4.0", optional = true }
 
 [dev-dependencies]
 arbitrary = "1.2"
-holochain = { version = "0.3.0-beta-dev.32", features = ["test_utils"] }
-rand = "0.7"
+holochain = { version = "0.3.0-beta-dev.38", features = ["test_utils"] }
+rand = "0.8"
 tokio = { version = "1.3", features = ["full"] }
-utilities = { path = "utilities" }
 kitsune_p2p_types = "0.3.0-beta-dev.14"
+
+[features]
+default = ["lair_signing", "client_signing"]
+
+lair_signing = [
```

Comment:
Can't promise this still works. I have tried to maintain behavior but there are no tests for it. This would be perfectly possible to write a unit test for but I don't need it to work and I don't have time right now.

If somebody (e.g. @guillemcordoba) needs it and it's not working, please let me know and I'll fix/test it